ddtrace/internal/datastreams/kafka.py

```diff
@@ -13,7 +13,7 @@
 INT_TYPES = (int,)
 
 
-def dsm_kafka_message_produce(instance, args, kwargs):
+def dsm_kafka_message_produce(instance, args, kwargs, is_serializing):
```

Comment:
When is this called with `is_serializing`?

Reply:
ah patch.py 142, got it! I wonder if there's an easier way to show these two things are connected or if that's something we'd want?

Reply:
Actually I guess it's pretty simple to search for `kafka.produce.start`.

Reply:
I've asked myself the same question in other contexts recently and at the moment I agree with you that Ctrl+F is good enough.